models/models.js

```diff
@@ -41,3 +41,36 @@ exports.selectUsers = () => {
     return rows;
   });
 };
+
+exports.selectArticles = () => {
+  return db
+    .query(
+      "SELECT articles.article_id, articles.title, articles.topic, articles.author, articles.created_at, articles.votes, (SELECT COUNT(*) FROM comments WHERE comments.article_id = articles.article_id) AS comment_count FROM articles;"
+    )
+    .then(({ rows }) => {
+      return rows;
+    });
+};
+
+exports.selectCommentsByArticleId = (article_id) => {
+  return db
+    .query("SELECT * FROM comments WHERE comments.article_id = $1;", [
+      article_id,
+    ])
+    .then(({ rows }) => {
+      return rows;
+    });
+};
+
+exports.selectArticleByIdForComments = async (article_id) => {
```

Comment:
It's unclear to me what this function is for. It this checking if the article exists? if so, do both db requests need to be run every time?